### PR TITLE
Py week1

### PR DESCRIPTION
**Pull request by jturi**

### CODE REVIEW

**Comment by jturi** [comment]
Total of 3 solutions added. Description fixed (Wrong exercise name of Sum of odd numbers modified to Numbers-in-strings-7-kyu)

**Comment by jturi** [comment]
0 byte commits fixed

File: src/phyton/datastructures/tribonacci.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+def tribonacci(signature, n):
+    len_sig = len(signature)
+    trib_list = signature
+
+    if n <= 3:
+        return trib_list[:n]
```
Comment (Binjaaa) [inline src/phyton/datastructures/tribonacci.py:6]: 👍

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (Binjaaa) [inline .gitignore:1]: We need to merge .gitignore files I think but in a later PR.

**Review by Binjaaa — APPROVED** [review]

**Pull request merged**